core/src/dunit/scala/io/snappydata/cluster/SplitClusterDUnitSecurityTest.scala

```diff
@@ -591,6 +596,44 @@ class SplitClusterDUnitSecurityTest(s: String)
     assertTableDeleted(() => {sns.catalog.refreshTable(smartRowTab1)}, smartRowTab1)
   }
 
+  def testSnappyJob(): Unit = {
+    // Create config file with credentials
+    val jobConfigFile = s"$snappyProductDir/conf/job.config"
+    writeToFile(s"-u $jdbcUser1:$jdbcUser1", jobConfigFile)
+
+    // Run snappy job with credentials in a config file
+    val job = s"$snappyProductDir/bin/snappy-job.sh submit --app-name CreatePartitionedRowTable " +
+        s"--class org.apache.spark.examples.snappydata.CreatePartitionedRowTable " +
+        s"--app-jar $snappyProductDir/examples/jars/quickstart.jar  " +
+        s"--passfile $jobConfigFile"
```

Comment:
--passfile is the option provided by snappy-job.sh to specify file path containing the user credentials. Internally, we pass this path as --config to the curl command.
--config is the option provided by curl to specify file path containing configs.

So --config is not visible to snappy-job.sh but --passfile is. We could rename --passfile to --config but I wanted it to be different as snappy-job users should just specify credentials in it and not any other config.
So the documentation will clearly mention what the content of this file should be, if security is enabled.